src/main/java/de/tum/in/www1/artemis/repository/ParticipantScoreRepository.java

```diff
@@ -53,6 +53,13 @@ public interface ParticipantScoreRepository extends JpaRepository<ParticipantSco
     @EntityGraph(type = LOAD, attributePaths = { "exercise", "lastResult", "lastRatedResult" })
     List<ParticipantScore> findAllByExercise(Exercise exercise);
 
+    @Query("""
+            SELECT p
+            FROM ParticipantScore p
+            WHERE p.exercise.course.id = :courseId
+            """)
```

Comment:
This is a lot of loaded data, as all scores of all participants of all exercises are loaded.

Reply:
Also the JavaDoc is missing.

Reply:
This appears to be unused @MaximilianAnzinger 
![CleanShot 2024-05-27 at 16 19 28](https://github.com/ls1intum/Artemis/assets/23171488/a2a78cb1-16c3-4dc9-a130-96223808631d)